@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -357,50 +368,10 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
             }
         }
 
-        for (const polyline of geometry) {
-            // Compute the world position of the untiled line and its distance to the origin of the
-            // line to properly join lines.
-            const untiledLine: number[] = [];
-            let lineDist = 0;
-            if (hasUntiledLines) {
-                this.m_decodeInfo.targetProjection.projectPoint(
-                    polyline.untiledPositions![0],
-                    tmpV3r
-                );
-                polyline.untiledPositions!.forEach(pos => {
-                    // Calculate the distance to the next unnormalized point.
-                    this.m_decodeInfo.targetProjection.projectPoint(pos, tmpV3);
-                    lineDist += tmpV3.distanceTo(tmpV3r);
-                    tmpV3r.copy(tmpV3);
-
-                    // Pushed the normalized point for line matching.
-                    this.m_decodeInfo.targetProjection.projectPoint(pos.normalized(), tmpV3);
-                    untiledLine.push(tmpV3.x, tmpV3.y, tmpV3.z, lineDist);
-                });
-            }
-
-            const line: number[] = [];
-            const lineUvs: number[] = [];
-            const lineOffsets: number[] = [];
-            polyline.positions.forEach(pos => {
-                webMercatorTile2TargetWorld(extents, this.m_decodeInfo, pos, tmpV3);
-                line.push(tmpV3.x, tmpV3.y, tmpV3.z);
-                if (computeTexCoords) {
-                    const { u, v } = computeTexCoords(pos, extents);
-                    lineUvs.push(u, v);
-                }
-                if (hasUntiledLines) {
-                    // Convert from local to world space.
-                    tmpV3.add(this.m_decodeInfo.center);
-
-                    // Find where in the [0...1] range relative to the line our current vertex lies.
-                    const offset = this.findRelativePositionInLine(tmpV3, untiledLine) / lineDist;
-                    lineOffsets.push(offset);
-                }
-            });
-            lines.push(line);
-            uvs.push(lineUvs);
-            offsets.push(lineOffsets);
+        if (this.m_enableLineClipping) {
+            this.convertAndCutLines(lines, uvs, offsets, geometry, extents, computeTexCoords);
```

Comment:
``convert`` is not really a term used in this context. Please use something along the lines of ``transform`` or ``project``.